src/cljs/main/broadfcui/page/style_guide.cljs

```diff
@@ -0,0 +1,225 @@
+    (map render-icon-sample (sort (keys icons/icon-keys)))]
+   [:p {} "Use them like this:"]
+   (create-code-block "(icons/icon {} :icon-name)")])
+
+(def ^:private components
+  [:section {}
+   [:h2 {:style {:marginBottom "0.5rem"}} "Components"]
+   (create-sub-head "Modals")
+   [:p {} "We have a lot of options for creating modals. On a fundamental level, you " [:em {} "could"]
+    " just use " (create-code-sample "modal/push-modal") ", but don't do that. You'd have to define the
+    modal from scratch, which would be awful. Instead, use " (create-code-sample "comps/push-message") " and
+    its ilk, which includes methods for quickly creating confirmation modals, alerts, etc."]
+   [:p {} " Any button that spawns a modal should have an ellipsis on the end of its label, "
+    [:em {} "unless"] " that modal is just an \"are you sure?\" confirmation."]
+   [:p {} "Modal titles, the labels of buttons that open them, and
+   their confirm buttons should share terminology as much as possible, specifically they should use
+   a consistent verb. For example, a button marked \"Delete\" could open a modal titled
+   \"Delete Method\" with a confirmation button labeled \"Delete\". Consistency is key."]
+
+   (create-sub-head "Infoboxes")
+   [:p {} "Observe, the ever-useful infobox:"
+    (common/render-info-box {:text "Can we be friends?"})]
+   (create-code-block "(common/render-info-box {:text \"Infobox text or element here.\"]})")
+   [:p {} "An infobox is an instance of a " (create-code-sample "common/FoundationDropdown") ", but it should
+   only be used when there's a contextual explanation to be displayed. The most interactivity inside
+   of it should be an external link."]
+
+   (create-sub-head "Tooltips")
+   [:p {} "There are basically two types of tooltips in FireCloud. In the data tables, every cell
+   has a title attribute, so hovering over them spawns a standard browser tooltip. The purpose of
+   these is to show the full text contents of a cell, regardless of its width."]
+   [:p {} "We also have Foundation's tooltips, "
+    [common/FoundationTooltip {:text "which look like this." :tooltip "Ooooh, aaaah."}]]
+   (create-code-block "[common/FoundationTooltip {:text \"Text or element with tooltip\" :tooltip \"Tooltip contents\"}]")
+   [:p {} "These are used for description or explanation. Generally, it's something the user could
+   figure out either from context or from clicking, but we want to be explicit."]])
+
+(react/defc Page
+  {:render
+   (fn [_]
+     [:div {:style {:padding "1rem"}}
+      [:style {}                                            ; Yeah, yeah, they're overrides, so sue me.
+       ".menu .active {background-color: #457fd2; color: white !important;}
+       .menu {padding: 0;}
+       .menu > li > span {display: block; padding: 0.7rem 1rem; line-height: 1;}
+       p {margin: 0.5rem 0;}
+       .CodeMirror {height: auto; font-family: Menlo, monospace; font-size: 12px;}"]
```

Comment:
Wow this is surprisingly clean. Instead of the `!important` modifier, could you add a class to the parent div and use that?